### DISCUSSION TITLE
Why does Budibase brand itself as an 'internal tool' solution, and what would it mean to make it 'external' or public facing?

### DISCUSSION THREAD

**Discussion by MiLandry** [Budibase/budibase#7253] [General]
Great tool by the way.

**Comment by melohagan** [comment]
Hi @MiLandry 

*From my own understanding:*

Internal tools include apps that make make handling your business activities easier such as:
- Admin related (HR, holiday tracking, etc)
- Data collection through forms (field management)
- Task management / Ticket tracker
- Internal dashboards

Budibase is primarily built with these types of applications in mind.

External / public facing apps would be more along the lines of:
- e-Commerce websites for customers to browse (e.g. Shopify)
- SaaS products

**Reply by mmalessa80** [reply]
Thanks for this explanation. I think for those of us working in Government the 'data collection through forms' is also a very important aspect for external facing apps. This is where I think granular security filtering for datasources might be of great importance. Is this something on the roadmap?
